nimare/tests/test_meta_ibma.py

```diff
@@ -111,17 +107,6 @@ def test_ibma_smoke(testdata_ibma, meta, meta_kwargs, corrector, corrector_kwarg
         assert cres.get_map("z").ndim == 3
 
 
-def test_ibma_with_custom_masker(testdata_ibma):
```

Comment:
+1 for deleting this test, but maybe replace it with another small one that verifies an exception is raised if a different masker is passed?